infinispan/src/test/java/org/hibernate/ogm/datastore/infinispan/test/dialect/impl/InfinispanDialectWithClusteredConfigurationTest.java

```diff
@@ -162,7 +162,7 @@ public void shouldApplyForEachTupleInClusteredMode() throws Exception {
 
 		// then
 		MyConsumer consumer = new MyConsumer();
-		dialect2.forEachTuple( consumer, keyMetadata );
+		dialect2.forEachTuple( consumer, null, keyMetadata );
```

Comment:
To be consistent with the rest of the test, this should  be `emptyTupleContext()`